### ISSUE TITLE
About "transaction input or output"

### ISSUE THREAD

**Issue by youngmtool** [ProgrammingBlockchain/ProgrammingBlockchain#93]
At this page:
https://github.com/youngmtool/ProgrammingBlockchain/blob/master/bitcoin_transfer/spend_your_coin.md

At this sentence:
If you want to send 0.5 BTC from a transaction input with 1 BTC you actually have to spend it all!

1.
Should the above sentence be replaced with this sentence?:
If you want to send 0.5 BTC from an output of the different transaction which holds 1 BTC, you actually have to spend it all!

**Comment by nopara73** [comment]
How about this one:  

If you want to send 0.5 BTC from an unspent output, which holds 1 BTC, you actually have to spend it all!

**Comment by NicolasDorier** [comment]
ACK @nopara73

**Comment by youngmtool** [comment]
Adding "unspent" makes more clear but don't you think "of the different" next to "output" in the right side makes the meaning also clear?

**Comment by nopara73** [comment]
@youngmtool 
It's confusing for me. Honestly I don't really understand what you mean by different transaction. Is what you mean is: "you must spend an utxo of a transaction, that's not the one you are building right now."?

**Comment by youngmtool** [comment]
Yes, with thinking again, I so far assumed the transaction we're making is already included in the transaction of the block which will be created.

So my intention was to distinguish the transaction which will be included in a new block and transaction for UTXO.

At the moment when we're writing a transaction, there is no transaction in a block yet. 

Thanks for answer.

**Comment by andrasfuchs** [comment]
I changed the wording to the one recommended by @nopara73 in my recent PR #104, so if it gets accepted this issue can be closed.

**Comment by youngmtool** [comment]
@andrasfuchs Okay. Thanks for informing me.
